tincr/io/design/tincr_checkpoints.tcl

```diff
@@ -127,10 +128,23 @@ proc ::tincr::write_placement_xdc {args} {
             puts $xdc "set_property BEL [get_property BEL $cell] \[get_cells \{[get_name $cell]\}\]"
             puts $xdc "set_property LOC [get_property LOC $cell] \[get_cells \{[get_name $cell]\}\]"
             puts $txt "BEL [get_property BEL $cell] [get_cells [get_name $cell]]"  
-            puts $txt "LOC [get_name $cell] [get_property LOC $cell] [get_property BEL $cell] [get_tile -of [get_sites -of $cell]]"
+
+	    # To match the xdlrc file generated from TINCR's "write_xdlrc", IOB sites that are bonded
+	    # have been updated to use the package pin name rather than the actual site name.  
+	    # The Vivado .xdc files have not been changed, only the txt files used for parsing a design into rapidSmith2
```

Comment:
I think that the logic for writing the RS2 txt files should be split out into a separate function for a few reasons:

1. It would reduce the amount of work that the function needs to do to generate the desired data
1. When you run write_tcp, 99% of the time you only want one of the two files - if they were split, it would be one file per function
1. Easier to maintain